### DISCUSSION TITLE
How about using Copilot for Pull Requests?

### DISCUSSION THREAD

**Discussion by ouchi2501** [pipe-cd/pipecd#4646] [Ideas]
Copilot for PRs would make it easier to create PRs.
Include markers in the PR's TEMPLATE to summarize the content

- Markers
  - copilot:all showcases all the different kinds of content in one go.
  - copilot:summary expands to a one-paragraph summary of the changes in the pull request.
  - copilot:walkthrough expands to a detailed list of changes, including links to the relevant pieces of code.
  - copilot:poem expands to a poem about the changes in the pull request.

https://githubnext.com/projects/copilot-for-pull-requests

Currently in Technical Preview, but I feel it's worth using.
I think it would be good to enroll it in the waitlist, but I'd like to hear your opinion.

**Comment by kentakozuka** [comment]
Sounds good 👍

**Comment by ffjlabo** [comment]
Thank you! I added pipe-cd/pipecd in the waitlist

**Comment by t-kikuc** [comment]
Any progress?
